Week9Serialization/Week9Serialization/Serialization Classes/BinarySerializer.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System.IO;
+using System.Runtime.Serialization.Formatters.Binary;
+
+namespace Week9Serialization
+{
+    public class BinarySerializer : ISerializer
+    {
+        public PersonList<T> Deserialize<T>(PersonList<T> list) where T : Person
+        {
+            BinaryFormatter formatter = new BinaryFormatter();
+            using (FileStream reader = File.OpenRead("_b-persons.txt"))
+            {
+                return formatter.Deserialize(reader) as PersonList<T>;
+            }
+        }
+
+        public void Serialize<T>(PersonList<T> list) where T : Person
```

Comment:
Your methods should have XML comments. The XML commentary for this method could be something like, "Serializes the specified <see cref="PersonleList{T}" /> into a binary representation and saves that representation to a file." That would make it really easy for someone to understand and use your API. (-0.4)